ts/card-info/Revlog.svelte

```diff
@@ -133,6 +136,7 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
 
 <style lang="scss">
     .revlog-table {
+        font-family: monospace;
```

Comment:
```suggestion
```

Going to remove this as it's holding up the PR. It's not a long-term no, but if you think it's worth pursuing, it might be worth doing a brief poll on the forums showing before/after to see if it's a change people are keen on.

Thanks Rumo!

Reply:
Sorry, that was completely unintentional! I just wanted to try it out and wasn't aware that I had actually committed it.
Now your last comment makes a lot more sense. 😅

Reply:
😄